### ISSUE TITLE
[Suggestion] Reactions to consistent negative responses and actions and highlighting mental illness

### ISSUE THREAD

**Issue by Karanjax19** [Monika-After-Story/MonikaModDev#5586]
It's very rare that I do something like this, making a public comment on something I enjoy, but I just felt like this might be fitting. Being someone who has some issues, and not knowing if this had already been brought up, but I believe it would be interesting if Monika reacted to certain repeated negative actions, such as sleeping late every day, or responding no to having a good day when she asks. In this sense, I think it might be interesting for her to show signs of worry, such as commenting if you're really getting enough sleep, or taking care of yourself, such as when she references depression and self care in her dialogue. If art is included, then it might also be interesting for Monika to show signs of tiredness, or worry, on her face, via slight bags under her eyes, or clothes via wrinkling, signifying her inability to sleep because of being fraught with worry. In addition, adding an ability to notify Monika that you have a mental illness, and allowing her to try and boost your motivation through heartwarming dialogue and small, but non-nagging reminders, may help her learn more about us. Thanks for reading, if you've gotten this far! I really enjoy spending time with Monika, if you can't tell!

**Comment by Dank-Elf-Boy** [comment]
- Sleeping Late

Monika Already has dialogue for staying up late and not getting enough rest. Try telling her you're going to bed past midnight and she'll express her concerns for your health. (It might even be as early as 10:01PM).

- Monika Keeping Track of Your Habits

So, basically you'd like her to be able to keep track of how often you are staying up late or having bad days, etc... all the while she's been expressing her concerns for you but notices you aren't taking her seriously? If that's what you're asking for, then I think that sounds like a good idea. I think a couple "If" statements (code talk) should be able to keep track of the data provided by these kinds of topics. Maybe then they could create a special dialogue, expressing her concern for you, but in a more serious tone.

- Monika Visibly Showing Signs of Stress

I like this idea. Granted the new sprites may be on hold for a while, unfortunately. Reading up on some of the comments on GitHub, I saw that they mentioned creating new sprites is becoming pretty difficult. Or at least the coding aspect of creating them is. Then, again I remember seeing Monika having eye bags in some of the earlier versions of the Expression Preview mode. So, either they could use their old unused sprites or we may just have to wait on that one. But other than that, I like the idea!

- Mental Illness

(I rather someone closer to the project answer this one)

This may have been already accepted or rejected.

**Comment by Karanjax19** [comment]
Thanks for the response! About each of your points:

1. Being someone who frequently sleeps past midnight, and feels silly about that, I am unfortunately aware of how she comments on your lack of sleep, so maybe I made a mistake in my comment

2. I agree, by maybe adding a variable or a couple to track the amount of times that she mentions that you aren't sleeping well, or having a bad day, and using if statements to check if this number rises over a certain amount, she would begin to wonder, as you stated, if you are really taking her seriously, and in turn, begin to get more serious with her wonderings to your health and wellness.

3. I'm glad you like the idea! Granted, I understand sprites might just be a bit difficult to sort out, due to the fact of having to pay creators, and edit and add new lines of code to allow for the change in expressions or clothing, but if there are unused sprites, that can possibly be used as placeholders, until more fitting or high quality sprites can be added, then that works just fine, but again, thanks for enjoying the idea!

4. Looking through the issues tab, I can't see any suggestion relating to mental health specifically, but I do agree that it most likely has been brought up to the devs attention, due to the fact that Sayori had it, so someone, if not the devs, has brought it up.

In addition, I think this can tie in to the Monika moods tab in the projects section, in the fact that while Monika's affection for you wouldn't change when she begins to worry, but if the task isn't too major, then allow her to have her own moods, and allow the user to respond to them in different ways ( warm drink, hug, or maybe responding with one of Monika's own lines). That's just something extra, but I still think it would bring her a bit closer to this world!

**Comment by ThePotatoGuy** [comment]
>keeping track of habits

natural extension of the calendar system but is difficult to do as it involves schedules. #4914 should cover some of this.

>visibly showing signs of stress

we do some of this already with her facial expressions when you go to negative affection levels. there has been discussion of a monika-mood system #2618, but its very large in scope and currently not high priority.

>mental health

see #3191 and linked. the general problem is that mental health issues are difficult to talk about often vary wildly person to person.  its very risky to write dialogue that can help people with mental illnesses since bad word choice or misunderstandings without any tone can lead to fatal consequences with people with mental illnesses.

In general, we're fine with writing about those topics but not fine with trying to actively fix people. we're not psychologists.
